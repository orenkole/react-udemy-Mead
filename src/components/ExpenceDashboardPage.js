import React from 'react';
import ConnectedExpenseList from './ExpensesList';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesList from './ExpensesList';

const ExpenceDashboardPage = () => (
  <div>
    <ExpenseListFilters />
    <ExpensesList />
  </div>
);

export default ExpenceDashboardPage;