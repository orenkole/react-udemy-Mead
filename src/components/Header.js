import React from 'react';
import {NavLink} from 'react-router-dom';
const Header = () => (
  <header>
    <h1>Expensify</h1>
    <NavLink exact={true} activeClassName="is-active" to="/">Dashboard</NavLink>
    <NavLink exact={true} activeClassName="is-active" to="/help">Help</NavLink>
    <NavLink exact={true} activeClassName="is-active" to="/create">Create Expense</NavLink>
  </header>
);
export default Header;