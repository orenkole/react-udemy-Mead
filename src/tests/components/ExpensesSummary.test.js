import React from 'react';
import {shallow} from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';

test('should render ExpensesSummary with 1 expense', () => {
  const wrapper = shallow(<ExpensesSummary
    expensesCoun={1}
    expensesTotal={235}
  />);
  expect(wrapper).toMatchSnapshot();
})

test('should render ExpensesSummary with multiple expenses', () => {
  const wrapper = shallow(<ExpensesSummary
    expensesCoun={23}
    expensesTotal={34543245645}
  />);
  expect(wrapper).toMatchSnapshot();
})