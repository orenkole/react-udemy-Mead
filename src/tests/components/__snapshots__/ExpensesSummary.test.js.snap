// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
     totalling 
    $345,432,456.45
  </h1>
</div>
`;
